broker-daemon/utils/create-http-server.js

```diff
@@ -30,24 +30,30 @@ function createHttpServer (protoPath, rpcAddress, { disableAuth = false, enableC
     app.use(corsMiddleware())
   }
 
+  // If the RPC address we use for daemon is set to a default route (0.0.0.0)
```

Comment:
I moved these lines to the top of the file, because we still want to mutate this address whether disableAuth is true or not